src/ViewHelpers/DisplayCoursesViewHelper.php

```diff
@@ -23,6 +23,9 @@ public static function createCoursesTableLoop(array $courses, array $trainers):
             $remote = $course->getRemote() == 1 ? '&#x2713;' : '&#x10102';
             $inPerson = $course->getInPerson() == 1 ? '&#x2713;' : '&#x10102';
             $trainersByCourse = self::filterCoursesByTrainers($trainers, $course->getId());
+            $spacesTaken = $course->getSpacesTaken() === null ? 'N/A' : $course->getSpacesTaken();
+            $totalAvailableSpaces =
+            $course->getTotalAvailableSpaces() === null ? 'N/A' : $course->getTotalAvailableSpaces();
```

Comment:
these 2 lines do the same thing, seems to me this could be a method called twice instead.

you could also consider putting this functionality into the entity itself, so rather than returning `null` the getting returns the N/A